### DISCUSSION TITLE
Question from NCP

### DISCUSSION THREAD

**Discussion by n0rs3** [tteck/Proxmox#945] [General]
Hello @tteck :wave: I hope everything is well with you? 

I wanted to ask you if it is alright with you if we reference your install script for LXD/LXC on Proxmox for NCP in the README & on the website, if that is alright with you? :pray:

**Comment by tteck** [comment]
Yes, that's fine with me.

**Reply by n0rs3** [reply]
Thank you :pray: I will add it in the README and on the Website :smile:
